src/DynamoMLDataPipeline/DynamoMLDataPipelineExtension.cs

```diff
@@ -24,15 +20,10 @@ public string UniqueId
 
         public string Name { get { return "DynamoMLDataPipelineExtension"; } }
 
-        public IEnumerable<IExtension> RequestedExtensions => throw new NotImplementedException();
-
-        public event Func<string, IExtension> RequestLoadExtension;
-        public event Action<IExtension> RequestAddExtension;
-
         public void Startup(StartupParams sp)
         {
             DynamoMLDataPipeline = new DynamoMLDataPipeline();
-            DynamoMLDataPipeline.AuthTokenProvider = (IOAuth2AccessTokenProvider)sp.AuthProvider;
+            DynamoMLDataPipeline.AuthProvider = (IDSDKManager)sp.AuthProvider;
```

Comment:
yeah so, this will throw in Revit and Civil.

Reply:
Using IOAuth2UserIDProvider interface now, will update the Revit PR to extend this interface.